test/cypress/integration/reference/reference.cy.test.js

```diff
@@ -49,10 +51,28 @@ describe('Reference page', () => {
     it('when a valid label is entered, should then go to the amount page', () => {
```

Comment:
this test name needs to be updated to reflect the new behaviour

Reply:
Good spot - I have re-structured this file